packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -324,6 +324,13 @@ export interface FileSystemProps {
    * @default ReplicationOverwriteProtection.ENABLED
    */
   readonly replicationOverwriteProtection?: ReplicationOverwriteProtection;
+
+  /**
+   * Replication configuration for the file system.
+   *
+   * @default - no replication
+   */
+  readonly replicationConfiguration?: ReplicationConfiguration;
```

Comment:
Why aren't we allowing multiple destinations? ([CFN declaration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-efs-filesystem-replicationconfiguration.html))

Reply:
CloudFormation supports an array of destinations, but the maximum number of elements is restricted to 1. Therefore, I did not implement support for multiple destinations. 

Should I add support for it to accommodate future expansion?

Reply:
Tricky question.
I guess we can keep it like this for now and deprecate it in the future in favor of an array of configurations if necessary.

Reply:
I've updated to receive `replicationConfiguration` as array.

Reply:
I'm sorry that I misunderstood your opinion. Which implementation will you adopt?